topics/synthetic-biology/slides/introduction.html

```diff
@@ -0,0 +1,209 @@
+
+.left[**Synthetic biology** provides novel strategies to perturb endogenous networks, enabling systems-level data to be obtained under a myriad of otherwise inaccessible cellular states.]
+
+.center[![Centered image](../images/system_synthetic_biology.jpg)]
+
+---
+
+## What do biological engineers engineer?  ... DNA
+
+## Bottom-up approach
+
+.center[![Centered image](../images/bottom-up-approach.png)]
+
+---
+
+## Synthetic biology: Concepts
+
+.left[- Break down the DNA into biological **parts** (e.g. promoters, terminators, ribosome binding sites, genes) that we can study in a living organism in the particular way that they appear.]
+
+.left[- Define **rules** and **models** which tries to predict how a particular stretch of DNA such as promoters, terminators and binding sites, are working for the cell.]
+
+.left[- Use **Standard** parts so that biologists from different laboratories can work together on the same parts.]
+
+---
+
+### Parts Registry
+
+.left[To design new systems, synthetic biologists will look to natural biological systems to find functional units of DNA **or** synthesize new ones that do not naturally exist. These functional units are tested and characterized and may become components in a biological device or system. We refer to them as **parts**.]
+
+.left[By simplifying and **standardizing** biology, the engineer can go on to use **parts** to create their biological systems without spending time researching each individual component or worrying about how to assemble them together.]
+
+.left[The goal of the [Registry](http://parts.igem.org/Main_Page) is to aid in this process by creating a library of **standard parts** that have been tested, characterized, and organized so that users can find what they need when developing new biological systems.]
+
+.left[Parts are **centralized** in repositories like [SynbioHub](https://synbiohub.org/) where DNA and protein designs can be uploaded. It also facilitates searching for information about existing useful parts and designs by combining data from a variety of sources.]
+
+---
+
+### Parts Registry
+
+.left[The **Registry** divides a plasmid into two separate components:]
+
+.pull-left[<img src="../images/part_backbone.png" width="200"/>]
+.pull-left[
+- **the part**
+    - A *biological part*: a natural nucleic acid sequence that encodes a definable biological function
+    - A *standard biological part*: a biological part that has been refined in order to conform to one or more defined technical standards
+]
+
+.pull-right[<img src="../images/biobricks_logo.png" width="200"/>]
+.pull-left[
+- **the plasmid backbone**
+    - All plasmids in Parts Registry are in a standard **BioBricks** format which is a standard for interchangable parts, developed with a view to building biological systems in living cells
+]
+
+---
+## Synthetic biology approaches
+
+.pull-right[<img src="../images/circuit.png" width="120"/>]
```

Comment:
you should be able to include images with markdown syntax here too, to specify the width, you should be able to do something like:


```
![alt text](path/to/image.png){: width="50%"}
```

(Not 100% sure if this also works in the slides)

Either way, please include alt text here as well (`alt="good description of image"` if you need to keep the html include)

That is the what is causing the last remaning linting errors, here and a handful more images in the slide deck

Thanks!